src/components/pacbio/PacbioTagEditModal.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div>
+    <b-button :id="'editTag'+this.tag.id" size="sm" @click="show" variant="outline-primary">Edit</b-button>
+
+    <b-modal
+      id="editTagModal"
+      ref="modal"
+      title="Edit Tag"      
+    >    
+
+      <b-form id="editTagForm" v-on:submit.prevent>
+        <b-form-group id="input-group-1"
+                      label="Sample tag: "
+                      label-for="input-1">
+                      <span>{{ this.tag.sample_name }}</span>
```

Comment:
Dont think you need span and maybe the sample name could go along side Sample tag so its would read in one line

Reply:
OK, changed so it reads Sample tag: {sample_name}